lib/stat_tracker.rb

```diff
@@ -15,7 +15,24 @@ def self.from_csv(locations)
     end
     new(new_locations)
   end
+  
+  def processed_teams_data(locations)
```

Comment:
Good start! Let's keep working on it after a break.

Reply:
After dinner we'll be better!